vmdb/app/controllers/application_controller.rb

```diff
@@ -972,27 +972,16 @@ def get_reports_menu(group=session[:group], tree_type="reports", mode="menu")
       #array of all reports if menu not configured
       @rep = MiqReport.all.sort_by { |r| [r.rpt_type, r.filename.to_s, r.name] }
       if tree_type == "timeline"
-        @rep.each do |r|
-          if r.timeline != nil
-            @data.push(r)
-          end
-        end
+        @data = @rep.reject { |r| r.timeline.nil? }
```

Comment:
Can this just be `@data = @rep.select { |r| r.timeline }` or even `@data = @rep.select(&:timeline)`